ci-operator/step-registry/optional-operators/ci/subscribe/optional-operators-ci-subscribe-commands.sh

```diff
@@ -37,7 +37,9 @@ else
     echo "OO_INSTALL_NAMESPACE is '$OO_INSTALL_NAMESPACE'"
 fi
 
-if [[ -n $NS_NAMESTANZA ]]; then
+echo "NS_NAMESTANZA is '$NS_NAMESTANZA'"
```

Comment:
That's strange. The length of the `NS_NAMESTANZA` value should be `0`, but it's not based on the logs below. And, seems like this update didn't take effect. @petr-muller Any ideas? Thanks! 
```console
OO_INSTALL_NAMESPACE is 'default'
/bin/bash: line 43: NS_NAMESTANZA: unbound variable
error: failed to execute wrapped command: exit status 1
```